### PR TITLE
Add idna recipe

### LINKED ISSUE #1328: Add idna
Packaging [idna]( https://github.com/kjd/idna ).

### PR DESCRIPTION
**Pull request by goanpeca**
Fixes #1328

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks @goanpeca. LGTM.

File: recipes/idna/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - idna
+
+about:
+  home: https://github.com/kjd/idna
+  license: BSD Like
```
Comment (goanpeca) [inline recipes/idna/meta.yaml]: This is what pypi said.. did not read it though... :-p

Comment (jakirkham) [inline recipes/idna/meta.yaml]: Yep, I looked at the license file and would generally agree. It was a BSD 3-Clause license with an additional (not advertising) clause similar in form to the HDF5 license. Though there are parts under other licenses like PSF 2 and some BSD like license for Unicode stuff. We could list those out, but "BSD Like" seems sufficient to cover those generally (at least to me).

We could ask others if they have a preference.

**Comment by goanpeca** [comment]
Merge @jakirkham :-) ?

**Comment by jakirkham** [comment]
LGTM. Just waiting for Travis CI to clear up a bit.

**Pull request merged**